packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -289,115 +289,7 @@ class ReorderableListView extends StatefulWidget {
   _ReorderableListViewState createState() => _ReorderableListViewState();
 }
 
-// This top-level state manages an Overlay that contains the list and
```

Comment:
This looks more drastic than it is. The previous implementation of _RerorderableListViewState wrapped  `_ReorderableListContent` with an overlay that it managed. This is no longer needed, so this just removes the `_ReorderableListContent` widget and replaces the _ReorderableListViewState with the contents of the old `_ReorderableListContentState` which is where the real list state implementation was.